# This patch exports `sixel_allocator_new` to the generated dll library for the windows target
# as bug fix. Upstream PR in https://github.com/saitoha/libsixel/pull/151

diff --git a/include/sixel.h.in b/include/sixel.h.in
index bcdb66c..cf61715 100644
--- a/include/sixel.h.in
+++ b/include/sixel.h.in
@@ -506,7 +506,7 @@ extern "C" {
 #endif
 
 /* create allocator object */
-SIXELSTATUS
+SIXELAPI SIXELSTATUS
 sixel_allocator_new(
     sixel_allocator_t   /* out */ **ppallocator,  /* allocator object to be created */
     sixel_malloc_t      /* in */  fn_malloc,      /* custom malloc() function */
